modules/payloads/singles/cmd/unix/bind_busybox_telnetd.rb

```diff
@@ -32,6 +32,7 @@ def initialize(info = {})
     register_options(
       [
         OptString.new('LOGIN_CMD', [true, 'Command telnetd will execute on connect', '/bin/sh']),
+        OptString.new('TELNETD_PATH', [true, 'Path to Telnetd', 'telnetd'])
```

Comment:
Why? JustAnda7 has already written the custom wording for each module. If they hadn't I'd agree that generic wording is fine, but this seems like an unnecessary change for no benefit. At worst this may create unnecessary confusion (unlikely).

"The path to telnetd[ executable]" is self-explanatory and explicit.

"The path to the command executable" less so. Users will have to infer what "the command" is from the default value. In a Windows context, this may be perceived as `command.com`, although that seems unlikely.

This format was chosen as it is inline with established wording for other payloads (`reverse_jjs` is the only payload which offers a path option IIRC).

Reply:
The reason for this request was due to inconsistent phrasing in the the unique version values that we felt was easier to deal with by making it a generic sentence. I will admit this was the `lazy` way of addressing this instead of marking each description.